packages/connector-edge/tsconfig.json

```diff
@@ -11,5 +11,8 @@
     "include": [
         "src/**/*.ts",
         "tests/**/*.ts"
+    ],
+    "references": [
+        { "path": "../hint" }
```

Comment:
I thinks that `utils-debugging-protocol-common` is also needed here.

Reply:
It's not listed in `dependencies` or `devDependencies` of `packages/connector-edge/package.json`.

Is there a way it can be included that I'm missing?

Reply:
> It's not listed in dependencies or devDependencies 

Then that is an issue, it should be hehehe.

Reply:
@antross can you add it to the package.json, it should be a dependency, like in `chrome-connector`

Reply:
Will do.